app/scripts/background.js

```diff
@@ -802,6 +805,14 @@ export function setupController(
             controller.approvalController.accept(id, false);
             break;
           ///: END:ONLY_INCLUDE_IN
+          ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+          case SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES.confirmAccountCreation:
```

Comment:
I would prefer these approval types to live in the above enum

Reply:
@hmalik88 I agree, I have made an [issue here](https://github.com/MetaMask/accounts-planning/issues/70) to add them to the `ApprovalType` enum and migrate all of this custom types. We will get to this after this merges.